package.json

```diff
@@ -93,7 +94,8 @@
     "test-android-instrumentation": "yarn run docker-build-android && yarn run test-android-run-instrumentation",
     "test-android-unit": "yarn run docker-build-android && yarn run test-android-run-unit",
     "test-android-e2e": "yarn run docker-build-android && yarn run test-android-run-e2e",
-    "test-ios": "./scripts/objc-test.sh test"
+    "test-ios": "./scripts/objc-test.sh test",
+    "test-typescript": "dtslint types"
```

Comment:
Is anything running these tests during build/pr?

Reply:
Not yet, will address this!

Reply:
I've added running dtslint as part of the `analyze_code` CircleCI tests